hermes-console-vue/json-server/server.js

```diff
@@ -0,0 +1,27 @@
+const topics = require('./topics.json');
+const subscriptions = require('./subscriptions.json');
+const routes = require('./routes.json');
+
+const jsonServer = require('json-server');
+const server = jsonServer.create();
+
+const middlewares = jsonServer.defaults();
+server.use(middlewares);
+
+const rewriter = jsonServer.rewriter(routes);
+server.use(rewriter);
+
+server.post('/query/topics', (req, res) => {
+  res.jsonp(topics);
+});
+
+server.post('/query/subscriptions', (req, res) => {
+  res.jsonp(subscriptions);
+});
+
+const router = jsonServer.router('json-server/db.json');
+server.use(router);
+
+server.listen(3000, () => {
```

Comment:
Have you checked that all endpoints from db.json works like we expect them to work after defining server on our own?

Reply:
yes, they do